smartcontactmanager/src/main/java/com/scm/smartcontactmanager/Service/ContactServiceImpl.java

```diff
@@ -0,0 +1,5 @@
+package com.scm.smartcontactmanager.Service;
+
+public class ContactServiceImpl implements ConatctService{
```

Comment:
Service class should be annotated with @Service

Reply:
fixed please review